locations/storefinders/localisr.py

```diff
@@ -83,3 +86,36 @@ def parse(self, response):
 
     def parse_item(self, item: Feature, location: dict, **kwargs):
         yield item
+
+    @staticmethod
+    def storefinder_exists(response: Response) -> bool:
+        if response.xpath('//script[contains(@src, "app.localisr.io/js/storelocator/widget.js")]'):
+            return True
+        if response.xpath('//iframe[contains(@src, "app.localisr.io/public/store-locator")]'):
+            return True
+        return False
+
+    @staticmethod
+    def extract_spider_attributes(response: Response) -> dict:
+        api_key = ""
+        localisr_widget_url = response.xpath(
+            '//script[contains(@src, "app.localisr.io/js/storelocator/widget.js")]/@src'
+        ).get()
+        if localisr_widget_url:
+            if extracted_key := re.search(r"key=(\w+)&", localisr_widget_url):
+                api_key = extracted_key.group(1)
+        if not api_key:
+            localisr_iframe_url = response.xpath(
+                '//iframe[contains(@src, "app.localisr.io/public/store-locator")]/@src'
+            ).get()
+            if localisr_iframe_url:
+                if extracted_key := re.search(r"requestToken=(\w+)", localisr_iframe_url):
+                    api_key = extracted_key.group(1)
+        # TODO: request localisr_url and extract from it:
```

Comment:
I tried
https://www.pillowtalk.com.au/ > Find a store
![image](https://github.com/alltheplaces/alltheplaces/assets/365751/3609cf91-1367-46ec-a6ac-16bd6745da7c)

but it doesn't look like this API call

Reply:
Currently localisr isn't enabled for this branch: https://github.com/alltheplaces/alltheplaces/pull/6526/files#diff-045e0bf251ad22781d641ca34a38bd5397f0ec700ec0231df536c0e433374808R207

I'll shortly get to implementing localisr properly.